Description: Allow linking with --as-needed
 icu-config includes libraries in --ldflags, but does provide a way to get at
 only the libraries.
Author: Stefano Rivera <stefanor@debian.org>
Bug-Debian: http://bugs.debian.org/674496
Last-Update: 2012-05-25

Index: an-1.0/Makefile
===================================================================
--- an-1.0.orig/Makefile	2012-05-16 19:29:42.303174501 +0100
+++ an-1.0/Makefile	2012-05-25 11:22:05.417866691 +0100
@@ -25,6 +25,7 @@
 CFLAGS += $(shell icu-config --cflags)
 CPPFLAGS += $(shell icu-config --cppflags) -D_BSD_SOURCE=1 -D_GNU_SOURCE=1
 LDFLAGS += $(shell icu-config --ldflags)
+LIBS += $(shell icu-config --ldflags-libsonly)
 
 BIN=an
 MAN=an.6
@@ -37,6 +38,7 @@
 $(OBJS): $(HEADERS)
 
 $(BIN):	$(OBJS)
+	$(CC) $(CFLAGS) $(LDFLAGS) $^ $(LIBS) -o $@
 
 $(MAN):
 
